crates/nargo/tests/test_data/tuples/src/main.nr

```diff
@@ -5,7 +5,18 @@ fn main(x: Field, y: Field) {
     constrain pair.0 == 1;
     constrain pair.1 == 0;
 
-    let (a, b) = (0, 1);
+    let (a, b) = if true { (0, 1) } else { (2, 3) };
     constrain a == 0;
     constrain b == 1;
+
+   let cc = if x as u32 <1 { [0,1] } else { [2,3] };
+   constrain cc[0] == 2;
```

Comment:
This tests arrays rather than tuples, maybe it should go in a different file

Reply:
I think it is fine, the test is still very small and comprehensive.
The more test files we have, the longer the testing time will be.

Reply:
Done